test/hotspot/jtreg/serviceability/jvmti/GetOwnedMonitorInfo/GetOwnedMonitorInfoTest.java

```diff
@@ -46,19 +46,40 @@ public class GetOwnedMonitorInfoTest {
         }
     }
 
+    private static native void jniMonitorEnter(Object obj);
     private static native int check();
     private static native boolean hasEventPosted();
 
+    private static void jniMonitorEnterAndLetObjectDie() {
```

Comment:
I can see it is convenient to just inject this test case in an existing test, but I'm not sure it is necessarily the right thing to do. Serviceability folk may have a stronger opinion.

Reply:
Yeah, I was thinking the same. Maybe @sspitsyn or @plummercj could give guidance here?

Reply:
This looks okay. I see no problem with it.